### PR TITLE
Add cursor pointer to list and sorted columns, fix firefox 63 bug and move login link to the right.

### PR DESCRIPTION
**Pull request by jmarrero**

### CODE REVIEW

**Comment by jmarrero** [comment]
yep, committed.

**Comment by jmarrero** [comment]
Also added a fix for the delete functionality on older versions of firefox, like the one QA uses (Firefox 63).

**Comment by jmarrero** [comment]
Per CCS-2909 I moved the Login to the right.

File: frontend/src/app/components/Chrome/Header/User.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import { Link } from "react-router-dom";
+
+export default class Brand extends Component {
```
Comment (benradey) [inline frontend/src/app/components/Chrome/Header/User.tsx]: Should this be `User` instead of `Brand`?

**Comment by benradey** [comment]
I notice also that we only get the finger pointer when hovering over the text of the row now, but we can still click the areas that give the normal pointer and get to the preview anyway. That is a trivial issue that is not worth fixing right now, but maybe it's something we can revisit in the future.

File: frontend/src/app/components/Chrome/Header/User.tsx (hunk 1 above)
Comment (jmarrero) [reply frontend/src/app/components/Chrome/Header/User.tsx]: yep this is what Michael talked about the explicit exports, I have changed it now to be explicit so this kind of typos don't happen in these two classes at least. Will work to do the same in all files eventually.

**Comment by jmarrero** [comment]
yeah the css parent child relationship is not strait forward for me yet. Ill try to fix this in another PR.

**Comment by jmarrero** [comment]
Fixed the checkbox thing. Now this includes changes for all your comments.

**Pull request merged**